include/vkwrap/vkwrap/render_pass.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include "common/vulkan_include.h"
+
+#include <range/v3/algorithm/copy.hpp>
+#include <range/v3/iterator/insert_iterators.hpp>
+#include <range/v3/range/concepts.hpp>
+#include <range/v3/range/conversion.hpp>
+#include <range/v3/view/all.hpp>
+#include <range/v3/view/iota.hpp>
+#include <range/v3/view/transform.hpp>
+
+#include <concepts>
+#include <vector>
+
+namespace vkwrap
+{
+
+class SimpleRenderPassBuilder
```

Comment:
Well I think you can call it just `RenderPassBuilder`, no need to lengthen the class name